docs/specs/ops.yml

```diff
@@ -269,3 +269,24 @@ repos:
   - files:
       ContentTemplate/schemas/TestData.schema.json: "{}"
 outputs:
+---
+# Support source_monikers for rediction
+repos:
+  https://github.com/redirection/source-monikers:
+  - files:
+      docfx.yml: |
+        monikerDefinition: monikerDefinition.json
+      .openpublishing.redirection.json: |
+        { "redirections": [{"source_path": "a.md", "source_monikers": ["netcore-1.0", "netcore-2.0"], "redirect_URL": "/"}] }
```

Comment:
consider rename `source_monikers` to `monikers`? This feels more aligned with treating redirection properties as YAML header.